gatsby/plugins/gatsby-source-cks/src/downloader/downloader.test.ts

```diff
@@ -0,0 +1,113 @@
+import {
+	SourceNodesArgs,
+	ActivityTracker,
+	ProgressActivityTracker,
+} from "gatsby";
+
+import { downloadAllData } from "./downloader";
+
+import {
+	getAllPartialTopics,
+	getFullTopicCached,
+	getChangesSince,
+} from "../api/api";
+import {
+	ApiTopicsResponse,
+	ApiPartialTopic,
+	ApiFullTopic,
+	ApiTopicChangeResponse,
+} from "../api/types";
+
+jest.mock("./../api/api", () => ({
+	getAllPartialTopics: jest.fn().mockResolvedValue({
+		topics: [],
+	}),
+	getChangesSince: jest.fn(),
+	getFullTopicCached: jest.fn(),
+}));
+
+// Typed API mocks for convenience
+const apiMocks = {
+	getAllPartialTopics: getAllPartialTopics as jest.Mock<
+		Promise<ApiTopicsResponse>
```

Comment:
having the last angle bracket on a newline is a bit odd. Line 35 looks formatted better `getAllPartialTopics as jest.Mock<Promise<ApiTopicsResponse>>,`

Reply:
This is prettier reformatting it so will probably just leave as is...